imap/structure_test.go

```
@@ -168,3 +168,20 @@ func TestParseMessage_GODT_2513(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, parsed)
 }
+
+func Fuzz_NewParsedMessage(f *testing.F) {
+	in_seed1, err_1 := os.ReadFile(filepath.Join("testdata", "rfc822.eml"))
+	in_seed2, err_2 := os.ReadFile(filepath.Join("testdata", "bounds.eml"))
+
+	if err_1 != nil || err_2 != nil {
+		return
```

It would be better if the test failed explictely rather than silently.
e.g.:

```
require.NoError(f, err1)
require.NoError(f, err2)
```